selfdrive/car/gm/values.py

```diff
@@ -89,36 +90,48 @@ class GMPlatformConfig(PlatformConfig):
   dbc_dict: DbcDict = field(default_factory=lambda: dbc_dict('gm_global_a_powertrain_generated', 'gm_global_a_object', chassis_dbc='gm_global_a_chassis'))
 
 
+@dataclass(frozen=True, kw_only=True)
+class GMCarSpecs(CarSpecs):
+  tireStiffnessFactor: float = 0.444
+  steerActuatorDelay: float = 0.1
+  minEnableSpeed: float = 5 * CV.KPH_TO_MS
+  minSteerSpeed: float = 10 * CV.KPH_TO_MS
```

Comment:
already defined

Reply:
These are listed here to set default values for the GM platform

Reply:
Oh sorry, I thought you were referring to the fields, not the class definition. Looks like an incorrect merge; will fix.